CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## 0.9.0
+
+* Changelog started, changes from previous release `0.8.0`:
+
+* Major breaking:
+  * Removed instances due to migration to `haskeline >= 0.8 && < 0.9`:
+    * `instance MonadException m => MonadException(StateT(HashMap FilePath NExprLoc) m)`
+    * `instance MonadException m => MonadException(Fix1T StandardTF m)`
+
+* Minor:
+  * Development package neetly exposes majority of `Nixpkgs haskell.lib` features
```

Comment:
Changelogs are mostly for IT specialists.

Our IT users are Haskell or Nix specialists ... profit.

You can basically open a `default.nix`, change keys how do you like it, and use simple `nix-build` to check that the project builds. I am interested is it affects `ghcid` or `HIE`, IDK at the moment.

Reply:
I remembered now, I already know that `default.nix` affects the `HIE` directly, at least in `all-HIEs` `NixOS` setup that I documented.

Reply:
With `build.sh` by its idea and the code. It should be possible to just export shell variables into `env` and run `build.sh` anywhere, and it would import the same options.